ui/components/institutional/custody-confirm-link-modal/custody-confirm-link-modal.js

```diff
@@ -30,12 +30,12 @@ import {
   Button,
   BUTTON_VARIANT,
   Modal,
-  ModalHeader,
-  ModalContent,
   ModalOverlay,
   Text,
   Box,
 } from '../../component-library';
+import { ModalContent } from '../../component-library/modal-content/deprecated';
+import { ModalHeader } from '../../component-library/modal-header/deprecated';
 
 const CustodyConfirmLink = ({ hideModal }) => {
   const t = useI18nContext();
```

Comment:
Screenshots showing no visual changes to UI

### Before
<img width="1512" alt="Screenshot 2023-12-10 at 3 20 06 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/963272cc-9967-45c6-8e21-83c2b942e36a">

### After
<img width="1512" alt="Screenshot 2023-12-10 at 3 20 19 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/76e692d7-a72d-4dfb-8fd1-76f32b491038">